src/datomic_toolbox/core.clj

```diff
@@ -229,3 +229,24 @@
         {:created-at (first tx-instants)
          :updated-at (last tx-instants)
          :timestamps tx-instants})))
+
+(defn retry-tx
```

Comment:
`retry-tx` sounds like it takes a transaction, like transaction data. But this is taking a function. Would this still work as just taking transaction data?